includes/class-amp-http.php

```diff
@@ -102,12 +102,16 @@ public static function send_header( $name, $value, $args = [] ) {
 	 *
 	 * @since 1.0
 	 *
+	 * @deprecated Use the `ServerTiming` service or its associated actions instead.
+	 *
 	 * @param string $name        Name.
 	 * @param float  $duration    Duration. If negative, will be added to microtime( true ). Optional.
 	 * @param string $description Description. Optional.
 	 * @return bool Return value of send_header call. If WP_DEBUG is not enabled or admin user (who can manage_options) is not logged-in, this will always return false.
 	 */
 	public static function send_server_timing( $name, $duration = null, $description = null ) {
+		_deprecated_function( 'AMP_HTTP::send_server_timing', '1.6', 'Use the AmpProject\AmpWp\Instrumentation\ServerTiming service or its associated actions instead.' );
```

Comment:
Would `__METHOD__` be better suited here?

```suggestion
		_deprecated_function( __METHOD__, '1.6', 'Use the AmpProject\AmpWp\Instrumentation\ServerTiming service or its associated actions instead.' );
```